docs/API.md

```diff
@@ -1154,8 +1158,9 @@ exact sending operation being done on a group.
 block (see `SRTO_SNDSYN` for details), so when this time is exceeded, it
 will behave as if in "non-blocking mode". The -1 value means no time limit.
 
-- Note that this flag set on a group is for group only, as it concerns the
-exact sending operation being done on a group.
+- Note that when this flag is set only on a group, it applies to a
+specific receiving operation being done on that group (i.e. it is not
```

Comment:
specific sending operation being done on that group (i.e. it is not

Reply:
Ah, my bad blind copy pasting.